composer.json

```diff
@@ -5,45 +5,48 @@
     "homepage": "https://github.com/silverstripe/recipe-kitchen-sink",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/recipe-plugin": "^1",
-        "silverstripe/installer": "4.x-dev",
-        "silverstripe/recipe-authoring-tools": "1.x-dev",
-        "silverstripe/recipe-blog": "1.x-dev",
-        "silverstripe/recipe-collaboration": "1.x-dev",
-        "silverstripe/recipe-form-building": "1.x-dev",
-        "silverstripe/recipe-reporting-tools": "1.x-dev",
-        "silverstripe/recipe-services": "1.x-dev",
-        "silverstripe/recipe-content-blocks": "2.x-dev",
-        "silverstripe/recipe-solr-search": "2.x-dev",
-        "silverstripe/recipe-ccl": "2.x-dev",
-        "silverstripe/developer-docs": "4.x-dev",
-        "silverstripe/registry": "2.x-dev",
-        "silverstripe/totp-authenticator": "4.x-dev",
-        "silverstripe/mfa": "4.x-dev",
-        "silverstripe/crontask": "2.x-dev",
-        "silverstripe/gridfieldqueuedexport": "2.x-dev",
-        "silverstripe/ldap": "1.x-dev",
-        "silverstripe/textextraction": "3.x-dev",
-        "silverstripe/realme": "4.x-dev",
-        "silverstripe/ckan-registry": "1.x-dev",
-        "silverstripe/webauthn-authenticator": "4.x-dev",
-        "silverstripe/security-extensions": "4.x-dev",
-        "silverstripe/subsites": "2.x-dev",
-        "silverstripe/html5": "2.x-dev",
-        "cwp/starter-theme": "3.x-dev",
-        "cwp/agency-extensions": "2.x-dev",
-        "cwp/watea-theme": "3.x-dev",
-        "cwp/cwp": "2.x-dev",
+        "php": "^8.1",
+        "silverstripe/recipe-plugin": "^2",
+        "silverstripe/installer": "5.x-dev",
+        "silverstripe/recipe-authoring-tools": "2.x-dev",
+        "silverstripe/recipe-blog": "2.x-dev",
+        "silverstripe/recipe-collaboration": "2.x-dev",
+        "silverstripe/recipe-form-building": "2.x-dev",
+        "silverstripe/recipe-reporting-tools": "2.x-dev",
+        "silverstripe/recipe-services": "2.x-dev",
+        "silverstripe/recipe-content-blocks": "3.x-dev",
+        "silverstripe/recipe-solr-search": "3.x-dev",
+        "silverstripe/recipe-ccl": "3.x-dev",
+
+        "silverstripe/developer-docs": "5.x-dev",
+        "silverstripe/registry": "3.x-dev",
+        "silverstripe/totp-authenticator": "5.x-dev",
+        "silverstripe/mfa": "5.x-dev",
+        "silverstripe/crontask": "3.x-dev",
+        "silverstripe/gridfieldqueuedexport": "3.x-dev",
+        "silverstripe/ldap": "2.x-dev",
+
+        "silverstripe/textextraction": "4.x-dev",
+        "silverstripe/realme": "5.x-dev",
+        "silverstripe/ckan-registry": "2.x-dev",
+        "silverstripe/webauthn-authenticator": "5.x-dev",
+        "silverstripe/security-extensions": "5.x-dev",
+        "silverstripe/subsites": "3.x-dev",
+        "silverstripe/html5": "3.x-dev",
```

Comment:
```suggestion
```

We're not going to need this module going forward.

https://github.com/silverstripe/silverstripe-framework/issues/10528

Reply:
DONE